### PR TITLE
Post item branch

### PR DESCRIPTION
**Pull request by wilsonwtcheng**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+class PostsController < ApplicationController
+
```
Comment (harrychenca) [inline app/controllers/posts_controller.rb]: The tab size is wrong

File: app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class Post < ActiveRecord::Base
+	# belongs_to :user
```
Comment (h8rry) [inline app/models/post.rb]: Tab size is wrong

File: db/migrate/20150519071341_add_userid_to_posts.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUseridToPosts < ActiveRecord::Migration
```
Comment (harrychenca) [inline db/migrate/20150519071341_add_userid_to_posts.rb]: This is not being used. If not, then we shouldn't have it.

File: app/views/posts/create.jbuilder [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (h8rry) [inline app/views/posts/create.jbuilder]: Let's not have line breaks if we don't need them

**Comment by h8rry** [comment]
Looks good :rose:

**Pull request merged**